packages/server/utils/analytics/analytics.ts

```diff
@@ -320,6 +321,16 @@ class Analytics {
     this.track(userId, 'Summary Email Setting Changed', subscribeToSummaryEmail)
   }
 
+  accountCreated = (userId: string, isInvited: boolean, isPatient0: boolean) => {
+    this.track(userId, 'Account Created', {
+      isInvited,
+      // properties below needed for Google Analytics goal setting
+      category: 'All',
+      label: 'isPatient0',
+      value: isPatient0 ? 1 : 0
```

Comment:
Looking at the [GA docs](https://support.google.com/analytics/answer/1033068#Anatomy&zippy=%2Cin-this-article), it's unclear whether this is actually how we should pass p0 information to GA. I.e. it sounds like `value` here is associated with the event, rather than the label. If I understand the GA docs correctly, we should actually only be sending `label`s that are either `isPatientZero` or `isNotPatientZero` (or similar).

Are we able to filter by both label and value in GA to do this analysis? What will the GA chart look like using this information?

Reply:
I think you are right! I was misled by the existing code. Thanks for pointing this out!